server/sandbox_stop_linux.go

```diff
@@ -135,6 +135,7 @@ func (s *Server) stopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 	if err := s.StorageRuntimeServer().StopContainer(sb.ID()); err != nil && errors.Cause(err) != storage.ErrContainerUnknown {
 		logrus.Warnf("failed to stop sandbox container in pod sandbox %s: %v", sb.ID(), err)
 	}
+	s.ContainerStateToDisk(podInfraContainer)
```

Comment:
The errcheck linter will complain that we do not check this error. We should print a warning if we don't want to fail here.

Reply:
Beside this, we have to add the following mock calls to `sandbox_stop_test.go` (line 40 and 67)

```
				ociRuntimeMock.EXPECT().UpdateContainerStatus(gomock.Any()).
					Return(nil),
```

Reply:
Fixed, thanks!